dsgrid_project/project.json5

```diff
@@ -556,19 +550,10 @@
         display_name: 'Transportation End Uses Collapsed',
         type: 'metric',
       },
```

Comment:
Since supplemental dimensions generally require mappings, should we list pairs of (supplemental dimension, mapping) instead of first listing all supplemental dimensions and then listing all mappings?

Reply:
https://github.com/dsgrid/dsgrid/issues/249